src/bomf/loader/entityloader.py

```diff
@@ -1,70 +1,63 @@
 """
 entity loaders load entities into the target system
 """
+
 import asyncio
 import json
 from abc import ABC, abstractmethod
 from datetime import datetime
 from pathlib import Path
 from typing import Awaitable, Callable, Generic, Optional, TypeVar
 
-import attrs
 from generics import get_filled_type
-from pydantic import BaseModel, TypeAdapter, ValidationError  # pylint:disable=no-name-in-module
+from pydantic import BaseModel, TypeAdapter, ConfigDict, Field, ValidationError  # pylint:disable=no-name-in-module
+from pydantic.dataclasses import dataclass
 
 _TargetEntity = TypeVar("_TargetEntity")
 
 
-@attrs.define(auto_attribs=True, kw_only=True)
+@dataclass(config=ConfigDict(arbitrary_types_allowed=True, validate_assignment=True))
 class EntityLoadingResult:  # pylint:disable=too-few-public-methods
     """
     Information gathered while loading a _TargetEntity into the target system.
     """
 
-    id_in_target_system: Optional[str] = attrs.field(
-        validator=attrs.validators.optional(attrs.validators.instance_of(str)), default=None
-    )
+    id_in_target_system: Optional[str]
```

Comment:
```suggestion
    id_in_target_system: Optional[str] = None
```
Den default-Wert musst du überall explizit mit angeben.